src/runtime_src/core/common/api/xrt_error.cpp

```diff
@@ -249,6 +249,9 @@ class error_impl
   std::string
   to_string()
   {
+    if (!m_errcode)
+      return std::string("No async error was detected");
```

Comment:
```suggestion
      return "No async error was detected";
```

Reply:
Removed std::string() and its test results remain the same.